Cargo.toml

```
@@ -25,17 +25,17 @@ git-version = "0.3"
 hidapi = "1.2"
 log = "0.4"
 object = { version = "0.24", default-features = false }
-probe-rs = "0.11"
-probe-rs-rtt = "0.11"
+probe-rs = { path = "../probe-rs/probe-rs" }
+probe-rs-rtt = { path = "../probe-rs/rtt" }
```

It looks like maybe these got changed to local paths for debugging but I'm wondering if the intention was to update them to `0.12`?

```suggestion
probe-rs = "0.12"
probe-rs-rtt = "0.12"
```